qa/tasks/cephfs/test_mirroring.py

```diff
@@ -1494,8 +1494,21 @@ def test_get_set_mirror_dirty_snap_id(self):
         """
         That get/set ceph.mirror.dirty_snap_id attribute succeeds in a remote filesystem.
         """
+        log.debug('reconfigure client auth caps')
```

Comment:
While this change is fine, the test is just validating the xattr set/get which is equally good to be done on the primary fs. But I agree, the test comment should match up with the test case :)

Reply:
ok, So what do you suggest? Change the test comment or this change is fine?

Reply:
I approved the PR -- LGTM.